### ISSUE TITLE
AttributeError: 'module' object has no attribute 'JSONDecodeError'

### ISSUE THREAD

**Issue by denniswisnia** [elnappo/ansible-role-check-mk-agent#8]
Hello Everybody,

I am getting the Error when I try to add the Hosts on a Ubuntu 18.04 Host: 
```

The full traceback is:
Traceback (most recent call last):
  File "<stdin>", line 113, in <module>
  File "<stdin>", line 105, in _ansiballz_main
  File "<stdin>", line 48, in invoke_module
  File "/tmp/ansible_check_mk_payload_bhPLzv/__main__.py", line 237, in <module>
  File "/tmp/ansible_check_mk_payload_bhPLzv/__main__.py", line 210, in main
  File "/tmp/ansible_check_mk_payload_bhPLzv/__main__.py", line 180, in host_exists
  File "/tmp/ansible_check_mk_payload_bhPLzv/__main__.py", line 153, in _api_request
AttributeError: 'module' object has no attribute 'JSONDecodeError'

fatal: [checkmk-prod-01]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"<stdin>\", line 113, in <module>\n  File \"<stdin>\", line 105, in _ansiballz_main\n  File \"<stdin>\", line 48, in invoke_module\n  File \"/tmp/ansible_check_mk_payload_bhPLzv/__main__.py\", line 237, in <module>\n  File \"/tmp/ansible_check_mk_payload_bhPLzv/__main__.py\", line 210, in main\n  File \"/tmp/ansible_check_mk_payload_bhPLzv/__main__.py\", line 180, in host_exists\n  File \"/tmp/ansible_check_mk_payload_bhPLzv/__main__.py\", line 153, in _api_request\nAttributeError: 'module' object has no attribute 'JSONDecodeError'\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
```

**Comment by jmvandry** [comment]
I have fixed this in the recent pr I made

**Comment by elnappo** [comment]
fixed in #9
